<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Apache Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="vLayLeftPannel" stretch="1,0,0">
      <item>
       <widget class="QTreeWidget" name="mainTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLeftNew">
        <property name="text">
         <string>新建</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLeftCheck">
        <property name="text">
         <string>检查语法</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="vLayRightPannel" stretch="0,0,0,0,0">
      <item>
       <widget class="QGroupBox" name="grbBasicSet">
        <property name="title">
         <string>基本设置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
         <item>
          <layout class="QFormLayout" name="fLayBasic">
           <item row="0" column="0">
            <widget class="QLabel" name="labWebsiteName">
             <property name="text">
              <string>网站名称</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edtWebsitName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labPort">
             <property name="text">
              <string>端口号</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edtPort">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labHostName">
             <property name="text">
              <string>主机名</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labRootDir">
             <property name="text">
              <string>根目录</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labMainLog">
             <property name="text">
              <string>主要日志</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="hLayRootDir">
             <item>
              <widget class="QLineEdit" name="edtRootDir">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRootDir">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spbHostName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>80</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="hLayMainLog">
             <item>
              <widget class="QLineEdit" name="edtMainLog">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnMainLog">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labErroLog">
             <property name="text">
              <string>错误日志</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="hLayErroLog">
             <item>
              <widget class="QLineEdit" name="edtErroLog">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnErroLog">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>24</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grbFlowSet">
        <property name="title">
         <string>流量设置</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="fLayFlow">
           <item row="0" column="0">
            <widget class="QLabel" name="labSpeedLimit">
             <property name="text">
              <string>启用限速</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labMaxFlow">
             <property name="text">
              <string>最大流量</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labMaxConnect">
             <property name="text">
              <string>最大连接</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="ckbSpeedLimit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string>0为不限制</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spbMaxFlow">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spbMaxConnect">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayRightBtns">
        <item>
         <widget class="QPushButton" name="btnRightSave">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRightCheck">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>检查语法</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labProgOutput">
        <property name="text">
         <string>程序输出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="edtProgOutput">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuApache_Manager">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <addaction name="menuApache_Manager"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
